GroversAlgorithm/ReferenceImplementation.qs

```diff
@@ -94,6 +95,10 @@ namespace Quantum.Kata.GroversAlgorithm {
         // Convert it into a phase-flip oracle and apply it
         let flipOracle = OracleConverter_Reference(allZerosOracle);
         flipOracle(register);
+        
+        // To fix the global phase difference, use the following line :
+        // For more details refer to the following Quantum SE question : https://quantumcomputing.stackexchange.com/questions/5973/counting-in-q-number-of-solutions/6446#6446
+        R(PauliI, 2.0 * PI(), register[0]);
```

Comment:
Can you add this line to `PhaseFlip_ControlledZ` as well? I believe that alternative solution suffers from the same issue

Reply:
Done